### PR TITLE
✨ amp-connatix-player: Add URL to iframe query params

### PR DESCRIPTION
**Pull request by gaborkalmar**
- added url to iframe query params because there was no way of accessing the full page url from the player

### CODE REVIEW

**Review by micajuine-ho — COMMENTED** [review]
This should be ok since this should not be the viewer URL. @alanorozco Does that sound right?

File: extensions/amp-connatix-player/0.1/amp-connatix-player.js [hunk 1]
```diff
@@ -271,6 +271,7 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     const urlParams = dict({
       'playerId': this.playerId_ || undefined,
       'mediaId': this.mediaId_ || undefined,
+      'url': this.getAmpDoc().getUrl()
```
Comment (alanorozco) [inline extensions/amp-connatix-player/0.1/amp-connatix-player.js:274]: This is unstable for the same document (URL may come from publisher's origin **or** a cache).

Consider using the AMP source URL instead. That way you always get the publisher's URL.

```suggestion
      'url': Services.documentInfoForDoc(this.element).sourceUrl,
```

Comment (gaborkalmar) [reply extensions/amp-connatix-player/0.1/amp-connatix-player.js]: @alanorozco fixed

File: extensions/amp-connatix-player/0.1/amp-connatix-player.js [hunk 2]
```diff
@@ -271,6 +271,7 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     const urlParams = dict({
       'playerId': this.playerId_ || undefined,
       'mediaId': this.mediaId_ || undefined,
+      'url': Services.documentInfoForDoc(element).sourceUrl
```
Comment (alanorozco) [inline extensions/amp-connatix-player/0.1/amp-connatix-player.js:274]: Looks like there's a formatting issue that won't let me merge:
```suggestion
      'url': Services.documentInfoForDoc(element).sourceUrl,
```

**Review by alanorozco — APPROVED** [review]

**Pull request merged**